src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -29,6 +29,8 @@ namespace Microsoft.ComponentDetection.Orchestrator;
 
 public class Orchestrator
 {
+    public const bool AutomaticallyProcessExperiments = false;
```

Comment:
This needs to belong in [`DetectorExperiments`](https://github.com/microsoft/component-detection/blob/4698b588479d3f7fdf9759666f4cbb30e28eb4ad/src/Microsoft.ComponentDetection.Orchestrator/Experiments/DetectorExperiments.cs)